src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java

```diff
@@ -1012,31 +1007,28 @@ public void testGetIdentifiablesCount() throws Exception {
         insertFilter(filterId3, hvdcLineFilter);
         checkFormFilter(filterId3, hvdcLineFilter);
 
-        CriteriaFilter substationFilter = new CriteriaFilter(
-                filterId4,
-                new Date(),
-                new SubstationFilter("NHV1_NHV2_4", "equipmentName_4", new TreeSet<>(Set.of("FR", "BE")), null)
-        );
-
-        MultiValueMap<String, String> params = new LinkedMultiValueMap<>();
-        params.add("networkUuid", NETWORK_UUID.toString());
-        params.add("variantId", VARIANT_ID_1);
-        final var json = TestUtils.resourceToString("/json/identifiables.json");
-        Map<String, List<Integer>> filtersComplexityCount = objectMapper.readValue(
-                mvc.perform(post("/" + FilterApi.API_VERSION + "/filters/identifiables-count")
+        MultiValueMap<String, String> params = new LinkedMultiValueMap<>(Map.of(
+                "networkUuid", List.of(NETWORK_UUID.toString()),
+                "variantId", List.of(VARIANT_ID_1),
+                "ids[g1]", List.of(filterId1.toString()),
+                "ids[g2]", List.of(filterId2.toString()),
+                "ids[g3]", List.of(filterId3.toString()),
+                "ids[g4]", List.of(UUID.randomUUID().toString())
+        ));
+        Map<String, Long> identifiablesCount = objectMapper.readValue(
+                mvc.perform(get("/" + FilterApi.API_VERSION + "/filters/identifiables-count")
                                 .params(params)
-                                .content(json)
                                 .contentType(APPLICATION_JSON))
                         .andExpect(status().isOk())
                         .andReturn().getResponse().getContentAsString(),
                 new TypeReference<>() {
                 });
-        assertEquals(1, filtersComplexityCount.get("0").size());
-        assertEquals(1, filtersComplexityCount.get("1").size());
-        assertEquals(1, filtersComplexityCount.get("2").size());
-        assertEquals(0, filtersComplexityCount.get("3").size());
+        assertEquals(1, identifiablesCount.get("g1").longValue());
+        assertEquals(1, identifiablesCount.get("g2").longValue());
+        assertEquals(0, identifiablesCount.get("g3").longValue());
```

Comment:
Why filter of filterId3 is not returning 1 anymore ?

Reply:
It's not the same
`filtersComplexityCount` was a list (List.of(0))
`identifiablesCount` is the value in the list

Reply:
indeed sorry ! :)